.github/workflows/pythonapp.yml

```diff
@@ -48,7 +48,7 @@ jobs:
         PLATALEA_DATA_ROOT: ${{ github.workspace }}/data/flickr1d
       run: |
         pip install sklearn python-Levenshtein coverage nltk
-        coverage run experiments/basic-stack/run.py --epochs=1 --meta=dataset.json --audio_features_fn=mfcc_features.pt
+        coverage run experiments/basic-default/run.py --epochs=1 --meta=dataset.json --audio_features_fn=mfcc_features.pt
```

Comment:
Fine by me, but why default instead of stack? Can we remove one or the other experiment?

Reply:
Ah I see you removed basic-stack below :)